lib/watir/elements/table_cell.rb

```diff
@@ -2,5 +2,12 @@ module Watir
   class TableCell < HTMLElement
     alias_method :colspan, :col_span
     alias_method :rowspan, :row_span
+
+    def column_header
+      index = self.previous_siblings.size
+      table = self.parent(tag_name: 'table')
+      TableHeaderCell.new(table.tr, {tag_name: 'th', index: index})
```

Comment:
This can be inconsistent with [Table#hashes](https://github.com/watir/watir/blob/master/lib/watir/elements/table.rb#L19-L22). `#hashes` checks for `td` and `th` elements, as well as makes sure the table has a basic layout.
~~~~~~~~ruby
cells = row.cells.to_a
if cells.length != headers.length
  raise Exception::Error, "row at index #{idx} has #{cells.length} cells, expected #{headers.length}"
end
~~~~~~~~

I don't have a strong opinion on which one is correct. But it would be nice to be consistent.